src/style.css

```diff
@@ -0,0 +1,105 @@
+html {
+  font-family: system-ui;
+}
+
+body {
+  background-color: rgb(243, 242, 242);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.todo-list {
+  width: 50%;
+  margin: auto;
+  height: auto;
+  background-color: #fff;
+  box-shadow: 0 0 5px 5px rgb(231, 231, 231);
+  padding: 10px;
+}
+
+h3 {
+  font-style: normal;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(233, 233, 233);
+}
+
+.add-task {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(233, 233, 233);
+}
+
+.input-task {
+  outline: 0;
+  border: 0;
+}
+
+.input-task::placeholder {
+  font-style: italic;
+  font-size: 14px;
+}
+
+.fa-arrow-left {
+  margin-right: 10px;
+  cursor: pointer;
+}
+
+.input-display {
+  border: 0;
+  outline: 0;
+  font-size: 16px;
+  cursor: auto;
+}
+
+.completed {
+  text-decoration: line-through;
+}
+
+.tasks-list {
+  list-style: none;
+  padding-left: 10px;
+}
+
+.tasks-list > li {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 10px;
+  border-bottom: 1px solid rgb(224, 224, 224);
+  padding-left: 0;
+}
+
+.task {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  gap: 10px;
+  margin-bottom: 10px;
+}
+
+.fa-trash-can {
+  cursor: pointer;
+}
+
+.footer {
+  background-color: rgb(243, 242, 242);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#clear-tasks {
+  border: 0;
+  outline: 0;
```

Comment:
I also advise that this border and outline property should be declared within a class and applied to all divs required or any other method you deem fit, like grouping all the classes around the same properties. 😃